Sources/_SubprocessCShims/include/target_conditionals.h

```diff
@@ -6,7 +6,6 @@
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
-// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
```

Comment:
Is this change intentional? Should we remove it from all headers?

Reply:
Yes